scripts.js

```diff
@@ -0,0 +1,107 @@
+// Select the hamburger icon and the navigation menu element from the DOM
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+const navLink = document.querySelectorAll('.nav-link');
+
+// Define the mobileMenu() that toggles the 'active' class on the hamburger icon and the nav menu
+function mobileMenu() {
```

Comment:
- [OPTIONAL] As suggested by the first reviewer, `To follow JavaScript best practices and keep up with global JavaScript best practices, ` you can use arrow function for your function declarations. You can change your function to `const mobileMenu = ( ) => {}` :+1: 
- You can also do this for other functions in this file like the `closeMenu and randomSpeaker` functions. :sparkle: